docs/changelog.md

```
@@ -5,6 +5,14 @@ description: The latest updates and changes to CodeRabbit.
 sidebar_position: 9
 ---
 
+## October 31, 2024
+
+### Learnings and Metrics for CodeRabbit Self-Hosted Customers
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)